lib/generators/double_entry/install/templates/migration.rb

```diff
@@ -54,9 +54,21 @@ def self.up
       t.text       "log"
       t.timestamps                             :null => false
     end
+
+    create_table "double_entry_line_metadata", :force => true do |t|
+      t.integer    "line_id",               :null => false
+      t.string     "key",     :limit => 48, :null => false
+      t.string     "value",   :limit => 64, :null => false
+      t.timestamps                          :null => false
+    end
+
+    add_index "double_entry_line_metadata", ["line_id"], :name => "lines_meta_line_id_idx"
+    add_index "double_entry_line_metadata", ["key"],     :name => "lines_meta_key_idx"
+    add_index "double_entry_line_metadata", ["value"],   :name => "lines_meta_value_idx"
```

Comment:
I take it back. I dunno how the whole left index thingamajig works in non-mysql dbs...